mParticle-BranchMetrics/MPKitBranchMetrics.m

```diff
@@ -95,9 +98,72 @@ - (MPKitExecStatus*_Nonnull)didFinishLaunchingWithConfiguration:(NSDictionary *)
     }
     self.forwardScreenViews = [configuration[ekBMAForwardScreenViews] boolValue];
     self.enableAppleSearchAds = [configuration[ekBMAEnableAppleSearchAds] boolValue];
+    [self updateIdentityType:configuration];
     return [self execStatus:MPKitReturnCodeSuccess];
 }
 
+- (void)updateIdentityType:(NSDictionary *)configuration {
+    //Shouldn't the config be passing a number rather than a string?
+        NSString *identityString = configuration[@"USER_IDENTIFICATION_TYPE"];
+        if (identityString != nil) {
+            if ([identityString isEqualToString:@"MPID"]) {
+                _isMpidIdentityType = true;
+            } else if ([identityString isEqualToString:@"customerid"]){
+                _identityType = MPIdentityCustomerId;
+            } else if ([identityString isEqualToString:@"email"]){
```

Comment:
Steve Hwang pointed this out, but we do specifically not want to use email as an identity type.